src/urllib3/contrib/pyopenssl.py

```diff
@@ -446,6 +446,8 @@ def set_ciphers(self, ciphers):
         self._ctx.set_cipher_list(ciphers)
 
     def load_verify_locations(self, cafile=None, capath=None, cadata=None):
+        if cadata is not None:
+            raise ssl.SSLError("CA data not supported in PyOpenSSL")
```

Comment:
Does it make more sense to use `NotImplementedError` here?

Reply:
Yes, I think you are right.
I used `ssl.SSLError` because we have something similar in 
https://github.com/urllib3/urllib3/blob/b83cc4698e9c74baab1ce17500f170e7a5170d07/src/urllib3/util/ssl_.py#L131